README.md

```diff
@@ -139,14 +139,10 @@ In our example above we use `$routerOnActivate` to fetch some accountDetails wit
 We use `$routerOnActivate` in favor of `$routerCanActivate` since we didn't find a way to pass data from the `$routerCanActivate` method to the controller instance. Another advantage is that we can specify `$routerOnActivate` as part of the controller whereas `$routerCanActivate` needs to be defined on the component definition (one thing that bugged us about the `$resolve`-approach too).
 
 ## Conclusion
-* took us a while to get started
-  * how to get the router
-  * found some bugs
-  * took them some time to fix
-  * now faster
-* fulfills our requirements
-  * CanReuse, OnActivate, OnReuse
-  * OnActivate -> resolve
-* Components
+It took us some time to get startet with the new ngComponentRouter. First of all we needed to compile the router from source which, even today, [does not](https://github.com/angular/angular/issues/7457) really [work on windows](https://github.com/angular/angular/issues/7457). Now you can easily install the npm package. Then we found some bugs and waited for about three to four weeks before we even got any feedback. Since petebacondarwin joined the router team this changed dramatically. Normally you get some feedback within 24 hours.
```

Comment:
> First of all we needed to compile the router from source which,

First of all we had to compile the ngComponentRouter by ourselves which,

> Now you can easily install the npm package

Since the angular team provides a npm package you can easily install it via npm install....

> Then we found some bugs and waited for about three to four weeks before we even got any feedback. Since petebacondarwin joined the router team this changed dramatically. Normally you get some feedback within 24 hours.

 While figuring out how the ngComponentRouter ought to work, we found some bugs and sent a few pull request which are mostly related to the angular-1 integration layer. Thanks to the angular team and especially to petebacondarwin, who took care of the angular 1.x ngComponentRouter lately All our PR's got merged by now and are integrated into the official npm package.